src/bin/bindgen.rs

```diff
@@ -91,6 +92,10 @@ Options:
                                   matching <regex>.  Same behavior on emptyness
                                   than the type whitelisting.
 
+    --bitfield-enum=<regex>       Mark a bitfield as being used as an enum.
```

Comment:
Nitpick: since this is a regex that presumably makes anything that matches it a bitfield enum (I haven't read the rest of the patch yet), we should rephrase this a little bit:

> Mark any union whose name matches `<regex>` as a set of bitfield flags instead of an enumeration. Bindgen will generate a type with bitwise arithmetic operators, instead of a Rust `enum`. If the union is anonymous, match `<regex>` against its members.

(Does this patch handle anonymous unions as bitfields?)

I also think it might make sense to rename this option to `--bitfield-union` or simply `--bitfield` since `enum` is purely on the Rust side of things.